src/main/resources/db/changelog/changes/v2.1.0/changelog-v2.1.0.xml

```diff
@@ -11,7 +11,7 @@
     </preConditions>
     <createView
       replaceIfExists="true"
-      viewName="src_vouchers">
```

Comment:
We probably need runOnChange=true for these changesets

Reply:
Yep, pretty much any time an existing changeset gets modified, we should probably add the runOnChange property. Otherwise, it will break in any dev envs that have already run the changeset